blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -1009,11 +1068,15 @@ def draw(self, context):
         col.enabled = preferences.terasology_module_directory != ""
         meshFileName = getTargetTerasologyMeshFileName(context)
         animFileName = getTargetTerasologyAnimFileName(context)
+        # value=context.scene.Scale
+        # value=ExportMD5.md5scale.Scale
+        # print("^^^^^^^^^^^",value)
+        # col.prop(ExportMD5.md5scale,"Scale")  

```

Comment:
commented out code: should be removed. Version history can keep track of old code versions.